python/cudf/cudf/_cuda/gpu.pyx

```diff
@@ -80,6 +59,229 @@ class CUDADriverError(RuntimeError):
+        <underlying_type_error> cudaError.cudaErrorTooManyPeers
+    cudaErrorHostMemoryAlreadyRegistered = \
+        <underlying_type_error> cudaError.cudaErrorHostMemoryAlreadyRegistered
+    cudaErrorHostMemoryNotRegistered = \
+        <underlying_type_error> cudaError.cudaErrorHostMemoryNotRegistered
+    cudaErrorHardwareStackError = \
+        <underlying_type_error> cudaError.cudaErrorHardwareStackError
+    cudaErrorIllegalInstruction = \
+        <underlying_type_error> cudaError.cudaErrorIllegalInstruction
+    cudaErrorMisalignedAddress = \
+        <underlying_type_error> cudaError.cudaErrorMisalignedAddress
+    cudaErrorInvalidAddressSpace = \
+        <underlying_type_error> cudaError.cudaErrorInvalidAddressSpace
+    cudaErrorInvalidPc = \
+        <underlying_type_error> cudaError.cudaErrorInvalidPc
+    cudaErrorLaunchFailure = \
+        <underlying_type_error> cudaError.cudaErrorLaunchFailure
+    cudaErrorCooperativeLaunchTooLarge = \
+        <underlying_type_error> cudaError.cudaErrorCooperativeLaunchTooLarge
+    cudaErrorNotPermitted = \
+        <underlying_type_error> cudaError.cudaErrorNotPermitted
+    cudaErrorNotSupported = \
+        <underlying_type_error> cudaError.cudaErrorNotSupported
+    cudaErrorSystemNotReady = \
+        <underlying_type_error> cudaError.cudaErrorSystemNotReady
+    cudaErrorSystemDriverMismatch = \
+        <underlying_type_error> cudaError.cudaErrorSystemDriverMismatch
+    cudaErrorCompatNotSupportedOnDevice = \
+        <underlying_type_error> cudaError.cudaErrorCompatNotSupportedOnDevice
+    cudaErrorStreamCaptureUnsupported = \
+        <underlying_type_error> cudaError.cudaErrorStreamCaptureUnsupported
+    cudaErrorStreamCaptureInvalidated = \
+        <underlying_type_error> cudaError.cudaErrorStreamCaptureInvalidated
+    cudaErrorStreamCaptureMerge = \
+        <underlying_type_error> cudaError.cudaErrorStreamCaptureMerge
+    cudaErrorStreamCaptureUnmatched = \
+        <underlying_type_error> cudaError.cudaErrorStreamCaptureUnmatched
+    cudaErrorStreamCaptureUnjoined = \
+        <underlying_type_error> cudaError.cudaErrorStreamCaptureUnjoined
+    cudaErrorStreamCaptureIsolation = \
+        <underlying_type_error> cudaError.cudaErrorStreamCaptureIsolation
+    cudaErrorStreamCaptureImplicit = \
+        <underlying_type_error> cudaError.cudaErrorStreamCaptureImplicit
+    cudaErrorCapturedEvent = \
+        <underlying_type_error> cudaError.cudaErrorCapturedEvent
+    cudaErrorStreamCaptureWrongThread = \
+        <underlying_type_error> cudaError.cudaErrorStreamCaptureWrongThread
+    cudaErrorTimeout = \
+        <underlying_type_error> cudaError.cudaErrorTimeout
+    cudaErrorGraphExecUpdateFailure = \
+        <underlying_type_error> cudaError.cudaErrorGraphExecUpdateFailure
+    cudaErrorUnknown = \
+        <underlying_type_error> cudaError.cudaErrorUnknown
+    cudaErrorApiFailureBase = \
+        <underlying_type_error> cudaError.cudaErrorApiFailureBase
```

Comment:
Instead of doing this can we just declare the enum extern as `cpdef` which automatically generates the Python enum class as well? This way we don't have to manually redefine things here.